services/121-service/migration/1713363871246-remove-startedRegistration-state.ts

```diff
@@ -0,0 +1,22 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class RemoveStartedRegistrationState1713363871246
+  implements MigrationInterface
+{
+  name = 'RemoveStartedRegistrationState1713363871246';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    console.time('RemoveStartedRegistrationState1713363871246');
+
+    await queryRunner.query(
+      `UPDATE "121-service"."registration"
+             SET "registrationStatus" = 'registered'
+             WHERE "registrationStatus" = 'startedRegistration'`,
```

Comment:
@tijsziere it seems that this change as-is might not be smart, because users in state `startedRegistration` are probably not compatible with `registered` state. Is it safe to change this so that we delete the PAs with`"registrationStatus" = 'startedRegistration'` instead?

Reply:
Confirmed that this is ok to change via DevOps